core/server/models/base/bulk-operations.js

```diff
@@ -41,13 +43,20 @@ async function insert(table, data) {
 }
 
 async function delChunkSequential(table, chunk, result) {
-    for (const record of chunk) {
+    for (const id of chunk) {
         try {
-            await db.knex(table).where('id', record).del();
+            await db.knex(table).where('id', id).del();
             result.successful += 1;
         } catch (err) {
-            result.errors.push(err);
-            result.unsuccessfulIds.push(record);
+            const importError = new errors.DataImportError({
```

Comment:
@allouis comparing to bulk _inserts_ we don't have special handling for whe the removing members fails. This is why such in-place handling made the most sense to me. Also, `DataImportError` was inspired by the error handling done in the regular ghost importer. 

The insert failures would be handled inside the main import loop though, with more specific error detection I think.